jenkinsfiles/RHEL_Check.groovy

```diff
@@ -193,7 +194,32 @@ pipeline {
                     }
                 }
 
+                stage('Check_code_changes') {
+                    steps {
+                        script {
+                            dir("./src") {
+                                def code_changes = sh(
+                                    script: """
```

Comment:
Instead of such multiline script consider the following one-liner:
```groovy
code_changed = sh(
    script: "(! git diff --name-only main | grep -v ^documentation)",
    returnStatus: true,
    label: "Code_changes"
)
```

Or with inverted logic
```groovy
only_doc_changed= sh(
    script: "git diff --name-only main | grep -v ^documentation",
    returnStatus: true,
    label: "Code_changes"
)
```

Reply:
It works incorrectly if there are changes in documentation and in code.

Reply:
Are you sure? Have you noticed `-v` key?

Reply:
Sorry didn't notice.